wiki/BBCode/pl.md

```diff
@@ -298,7 +298,7 @@ Dodaje obraz z łącza.
 
 Jeżeli obraz nie znajduje się pod danym adresem, załączenie go nie będzie możliwe.
 
-**Nie dodawaj obrazów, używając lokalnej ścieżki dostępu!** Wskazanie `C:\Użytkownicy\[nazwa]\Obrazy\obraz.jpg` (lub innej podobnej) **nie zadziała**
+**Nie dodawaj obrazów, używając lokalnej ścieżki dostępu!** Wskazanie `C:\Użytkownicy[nazwa]\Obrazy\obraz.jpg` (lub innej podobnej) **nie zadziała**
```

Comment:
Not sure if this is the only case but this looks to be incorrectly applied.

Reply:
checked through ~200 files from diff (everything except tournament articles but they should be okay anyways) and found nothing wrong except that one

PR should be fine now i guess